test/helpers/protocolGateway.forcewait.js

```diff
@@ -0,0 +1,26 @@
+function forceWaitOnPublish(pg) {
```

Comment:
Missing license header

Reply:
It's there - this might be the refreshing problem again. You can check the latest version of the branch